internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestHybrid.java

```diff
@@ -1065,10 +1068,23 @@ public void testHybridMultipleVersions() throws Exception {
       }
       producer.stop();
 
+      Properties onlineProducerProps = new Properties();
+      try (VeniceProducer veniceOnlineProducer = OnlineProducerFactory.createProducer(
+          ClientConfig.defaultGenericClientConfig(storeName).setVeniceURL(cluster.getRandomRouterURL()),
+          new VeniceProperties(onlineProducerProps),
```

Comment:
I think we do this kind of stuff fairly often. Maybe we should have a static helped function so we can just do `VeniceProperties.empty()`...